gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -658,19 +664,27 @@ def validate(self, parsed_args, client=None):
     if not bigquery_util.table_exist(client, project_id, dataset_id, table_id):
       raise ValueError('Table {}:{}.{} does not exist.'.format(
           project_id, dataset_id, table_id))
-    if table_id.count(TABLE_SUFFIX_SEPARATOR) != 1:
-      raise ValueError(
-          'Input table {} is malformed - exactly one suffix separator "{}" is '
-          'required'.format(parsed_args.input_table,
-                            TABLE_SUFFIX_SEPARATOR))
-    base_table_id = table_id[:table_id.find(TABLE_SUFFIX_SEPARATOR)]
-    sample_table_id = bigquery_util.compose_table_name(base_table_id,
-                                                       SAMPLE_INFO_TABLE_SUFFIX)
-
-    if not bigquery_util.table_exist(client, project_id, dataset_id,
-                                     sample_table_id):
+    if parsed_args.custom_sample_info_table:
+      sample_project_id, sample_dataset_id, sample_table_id = (
+          bigquery_util.parse_table_reference(
+              parsed_args.custom_sample_info_table))
+    else:
+      sample_project_id = project_id
+      sample_dataset_id = dataset_id
```

Comment:
nit: Let's move these two lines to where `sample_table_id` is initiated.

Reply:
Done.